sprite.go

```diff
@@ -1331,15 +1317,11 @@ func (p *SpriteImpl) TurnTo__1(sprite SpriteName) {
 	p.turnTo(sprite)
 }
 
-func (p *SpriteImpl) TurnTo__2(degree float64) {
-	p.turnTo(degree)
-}
-
-func (p *SpriteImpl) TurnTo__3(dir specialDir) {
+func (p *SpriteImpl) TurnTo__2(dir Direction) {
 	p.turnTo(dir)
 }
 
-func (p *SpriteImpl) TurnTo__4(obj specialObj) {
+func (p *SpriteImpl) TurnTo__3(obj specialObj) {
 	p.turnTo(obj)
 }
 func (p *SpriteImpl) SetHeading(dir float64) {
```

Comment:
We need to also update type for `Sprite.Heading`、`Sprite.SetHeading`、`Sprite.ChangeHeading`、`TurningInfo.Dir` to keep consistent.